path_role.go

```diff
@@ -364,6 +380,10 @@ type roleStorageEntry struct {
 	// role.
 	ServiceAccountNamespaces []string `json:"bound_service_account_namespaces" mapstructure:"bound_service_account_namespaces" structs:"bound_service_account_namespaces"`
 
+	// ServiceAccountNamespaceSelector is the label selector string of the
+	// namespaces able to access this role.
+	ServiceAccountNamespaceSelector string `json:"bound_service_account_namespace_selector" mapstructure:"bound_service_account_namespace_selector" structs:"bound_service_account_namespace_selector"`
```

Comment:
Are we still using `structs`, and  `mapstructure` we can probably clean that up in a subsequent PR.

Reply:
I confirm we are still using `structs` and `mapstructure` in other places.